tools/esp8266/hmInverter.h

```diff
@@ -275,9 +275,9 @@ class Inverter {
                 else if (rec->assign == SystemConfigParaAssignment) {
                     DPRINTLN(DBG_DEBUG, "add config");
                     // get at least the firmware version and save it to the inverter object
-                    if (getPosByChFld(0, FLD_ACT_PWR_LIMIT, rec) == pos){
+                    if (getPosByChFld(0, FLD_ACT_ACTIVE_PWR_LIMIT, rec) == pos){
                         actPowerLimit = rec->record[pos];
-                        DPRINT(DBG_DEBUG, F("Inverter actual power limit: ") + String(actPowerLimit));
+                        DPRINT(DBG_DEBUG, F("Inverter actual power limit: ") + String(actPowerLimit, 1));
```

Comment:
Is this the rounding ?

Reply:
Nicht perfekt, aber ausreichend und mehr Info als vorher. Ist float mit einer Nachkommastelle, da round bisher nur in web enthalten ist.